### PR TITLE
Feat/db models

### PR DESCRIPTION
**Pull request by SpicyI**
this Pr:
-implements the DataBase models for Users, comments, subtitles, videos

### CODE REVIEW

File: backend/main_app/settings.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 SECRET_KEY = "django-insecure-ebwn=qk4lm)p3*l7&qh_+^%8_1c7_i6^n-x1g=_3*dxyqi6#ig"
```
Comment (hmoumani) [inline backend/main_app/settings.py:24]: we definitely should make this an env var ☠️

**Review by hmoumani — CHANGES_REQUESTED** [review]
GG

File: backend/main_app/settings.py (hunk 1 above)
Comment (SpicyI) [reply backend/main_app/settings.py:24]: but no secrets should be between us 😔

Comment (hmoumani) [reply backend/main_app/settings.py:24]: @Hattaki-Hajar

**Review by hmoumani — APPROVED** [review]
GG

File: backend/main_app/settings.py [hunk 2]
```diff
@@ -21,10 +21,10 @@
 # See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-ebwn=qk4lm)p3*l7&qh_+^%8_1c7_i6^n-x1g=_3*dxyqi6#ig"
+SECRET_KEY = os.getenv("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv("Django_DEBUG", "Flase") == "True"
```
Comment (Hattaki-Hajar) [inline backend/main_app/settings.py:27]: change to "False"

**Review by Hattaki-Hajar — CHANGES_REQUESTED** [review]
good job

File: backend/main_app/settings.py (hunk 2 above)
Comment (SpicyI) [reply backend/main_app/settings.py:27]: nice catch

**Review by Hattaki-Hajar — APPROVED** [review]

**Pull request merged**